packages/buildbucket-dart/tool/regenerate.sh

```diff
@@ -5,14 +5,24 @@
 
 set -ex
 
-mkdir -p lib/src/generated
+NO_CLONE=1
+if [[ $1 == '-no_clone' ]]; then
```

Comment:
What is the use case for `no clone`? Do we need to regenerate the whole package each time?

Reply:
Ah I added this to save myself some time when trying to build the protos. Instead of blowing away the directory every time where nothing has changed I just added the flag so it would not reclone all of the code. If you run it as intended then it won't do anything. I also did not want to build out a full on args parser just for this but I can if that is preferred.

`sh tool/regenerate.sh` will do everything
`sh tool/regenerate.sh -no_clone` will not blow away the dir and clone everything. This is much faster when you are trying to add variables. But I get this might not be needed so I am fine with removing if we don't.

Reply:
Actually. I will change it back.